indico/modules/attachments/templates/_attachments.html

```diff
@@ -41,11 +41,18 @@
                         {% trans name=attachment.title %}Edit file &quot;{{ name }}&quot;{% endtrans %}
                     {% else %}
                         {% trans name=attachment.title %}Edit link &quot;{{ name }}&quot;{% endtrans %}
-                    {% endif -%}"></a>
+                    {% endif -%}"
+                    title="
+                    {%- if attachment.type.name == 'file' -%}
+                        {% trans name=attachment.title %}Edit file &quot;{{ name }}&quot;{% endtrans %}
+                    {% else %}
+                        {% trans name=attachment.title %}Edit link &quot;{{ name }}&quot;{% endtrans %}
+                    {% endif -%}"></a>></a>
```

Comment:
that looks wrong...

Reply:
BTW if you want to do something more fancy that avoids duplication, you could rename `data-title` to `title` and add `data-title` without a value, and adapt the JS that opens these dialogs to use the `title` if `data-title` is present (not undefined) but empty